src/pages/NewResetPassword.tsx

```diff
@@ -0,0 +1,100 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import React, { useState } from 'react'
+import { Button, Container, Form } from 'react-bootstrap'
+import useTranslationHook from '../hook/TranslationHook'
+import { useNavigate, useParams } from 'react-router-dom'
+import { confirmResetPassword } from '../services/api'
+
+const NewResetPassword = (): JSX.Element => {
+  const [newPassword, setNewPassword] = useState<string>('')
+  const [confirmNewPassword, setConfirmNewPassword] = useState<string>('')
+  const [passwordError, setPasswordError] = useState<string>('')
+  const { t } = useTranslationHook()
+  const { uid, token } = useParams<string>()
+  const navigate = useNavigate()
+
+  const checkPasswordsMatch = (): boolean => {
+    return newPassword === confirmNewPassword
+  }
+
+  const onNewPasswordChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setNewPassword(e.target.value)
+    setPasswordError('')
+  }
+
+  const onConfirmNewPasswordChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    setConfirmNewPassword(e.target.value)
+    setPasswordError('')
+  }
+
+  const handleResetPasswort = async (e: React.MouseEvent<HTMLElement>): Promise<void> => {
+    try {
+      const isUidTokenValid = uid !== null && uid !== undefined && token !== null && token !== undefined
+
+      if (isUidTokenValid) {
+        if (checkPasswordsMatch()) {
+          await confirmResetPassword(uid, token, newPassword)
+          navigate('/login')
+        } else {
+          setPasswordError('Passwords don\'t match')
+          console.error('Passwords don\'t match')
```

Comment:
Don't you want to translate?

Reply:
done already
you looked too early my PR